recipe-server/client/control_new/components/common/forms.js

```diff
@@ -0,0 +1,242 @@
+import { Form, message } from 'antd';
+import autobind from 'autobind-decorator';
+import get from 'lodash.get';
+import pt from 'prop-types';
```

Comment:
nit: we've been using the full form `PropTypes` everywhere else. we should stay consistent.